recipes/paramz/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "paramz" %}
+{% set version = "0.7.4" %}
+{% set sha256 = "d60a6449ce238dae6183e87a0f203077c8a463315c17f267e065b173bf5795e4" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
+    - six
+    - decorator
```

Comment:
Perhaps add version restriction to `decorator` since this release appears to be fairly new (in the last year) and it's a stated requirement. 

https://github.com/sods/paramz/blob/master/setup.py#L119